src/routes/safe/components/Apps/components/ConfirmTxModal/ReviewConfirm.tsx

```diff
@@ -255,7 +257,7 @@ export const ReviewConfirm = ({
               cancelButtonProps={{ onClick: handleTxRejection }}
               confirmButtonProps={{
                 onClick: () => confirmTransactions(txParameters),
-                disabled: areTxsMalformed,
+                disabled: !isOwner || areTxsMalformed,
```

Comment:
Do we have any message for this, so that users are aware why this is disabled?

Reply:
Now we do 😄